Source/Core/Core/PowerPC/JitCommon/DivUtils.h

```diff
@@ -0,0 +1,15 @@
+// Copyright 2021 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include "Common/CommonTypes.h"
+
+struct Magic
```

Comment:
Maybe place the contents of this file within a namespace like JitCommon, given `Magic` is quite a generic name.